### PR TITLE
English translation

### PR DESCRIPTION
**Pull request by lebond**

### CODE REVIEW

**Comment by lebond** [comment]
Possible to add ingredient list

Ingredients
½ Package of Yeast
4.5dl Water (heated to 37 ℃)
0.5dl Olive Oil
½ Tablespoon of Salt (without Iodine)

File: README_en-GB.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+Recipe for bread baked in Dutch oven.
+
+## Act 1: _Make the doughl_
+1. Mix ½ package of yeast with 0.5dl of water which has been heated to 37 °C. Make sure the yeast spreads out.
+2. Add 4dl of 37 ℃ water to the yeast and whisk in 1dl of flour. Let the mixture take it easy while you watch an episode of Adventure Time.
+3. Add 0.5dl of olive oil and ½ tablespoon of salt without iodine (It's possible _with_ iodine but it's better without. Iodine kills).
+4. Slowly add flour to the mixture until your have a dough. Stop once the dough begins to release from the edges of the bowl (Sorry, I can’t be more specific than that. Flour is hard to measure).
+5. Cover with cling film and let the dough chillax in the fridge overnight.
+
+## Act 2: _Time to wake up, little friend_
```
Comment (lebond) [inline README_en-GB.md:11]: Northern British alternatives for "little friend"
pet
sausage
duck

File: README_en-GB.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+>It’s the morning and it’s early. The kids want to watch Bolibompa but Women's Prison isn’t finished yet
```
Comment (lebond) [inline README_en-GB.md:12]: There is perhaps a better known prison TV show title?

File: README_en-GB.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+
+1. Remove the dough from the fridge and place it on a well floured surface. _Fold_ the dough like an envelope multiple times, adding extra flour so the texture becomes firm. The dough should reform after you poke it with a finger. Dust a chopping board with flour and place the dough on it.
+2. Make some coffee. Put the kids in front of Bolibompa. Explain that the bread isn’t ready yet. 
+3. Pre-heat the oven to approximately 250 °C. Place an iron pot, including it’s lid, in the oven.
+4. After 30-45 minutes remove the pot, take off the lid and add a small amount of polenta (the type where the corn does something sånt där majsjox spanjorer äter), alternatively a small amount of flour (which will most likely work just as well).
```
Comment (lebond) [inline README_en-GB.md:17]: remaining to translate

File: README_en-GB.md [hunk 4]
```diff
@@ -0,0 +1,27 @@
+# Dutch oven bread 
+Recipe for bread baked in Dutch oven.
+
+## Act 1: _Make the doughl_
```
Comment (lebond) [inline README_en-GB.md:4]: extra `l` to be removed

File: README_en-GB.md [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+2. Make some coffee. Put the kids in front of Bolibompa. Explain that the bread isn’t ready yet. 
+3. Pre-heat the oven to approximately 250 °C. Place an iron pot, including it’s lid, in the oven.
+4. After 30-45 minutes remove the pot, take off the lid and add a small amount of polenta (the type where the corn does something sånt där majsjox spanjorer äter), alternatively a small amount of flour (which will most likely work just as well). 
+
+## Act 3: _This is for your own good_
+1. Check out how gorgeous your dough looks! Using a sharp knife, cut a few cuts along the bread surface. Don’t be scared, it’s for the breads own good.
+2. Scrap the bread from the chopping board into the pot(help it with a spachelor). It's going to look silly. You are going to think that something has gone wrong. It hasn't. 
+3. Put the lid back on the pot and place all the goodness in the oven for 30 minutes
+4. Open the oven. Remove the lid and give the bread 10 more minutes. Your might think that the surface will be too hard. It won’t be. 
+5. Remove the pot from the over. Remove the bread from the pan and place on a rest.
```
Comment (lebond) [inline README_en-GB.md:24]: "Remove the bread from the pan" should be "Remove the bread from the pot"

File: README_en-GB.md (hunk 2 above)
Comment (jnystromdesign) [reply README_en-GB.md:12]: Or perhaps just go for "Seinfield re-runs". :)

**Pull request merged**